src/browser/modules/D3Visualization/components/DetailsPane.tsx

```diff
@@ -39,17 +40,54 @@ import { StyleableRelType } from './StyleableRelType'
 import { upperFirst } from 'services/utils'
 import { ShowMoreOrAll } from 'browser-components/ShowMoreOrAll/ShowMoreOrAll'
 
+export const MAX_LENGTH_SMALL = 150
+export const MAX_LENGTH_LARGE = 300
+type ExpandableValueProps = {
+  value: string
+  type: string
+  wideMode: boolean
+}
+function ExpandableValue({ value, wideMode }: ExpandableValueProps) {
+  const [expanded, setExpanded] = useState(false)
+
+  const maxLength = wideMode ? MAX_LENGTH_LARGE : MAX_LENGTH_SMALL
+
+  const handleExpandClick = () => {
+    setExpanded(true)
+  }
+
+  let valueShown = expanded
+    ? value
+    : value.slice(0, Math.min(maxLength, value.length))
+  const valueIsTrimmed = valueShown.length !== value.length
+  valueShown += valueIsTrimmed ? '...' : ''
```

Comment:
There is a unicode character for elipsis we could use here "…" not sure if it's preferred, but it's an option at least!

Reply:
Sure!